.vscode/settings.json

```diff
@@ -0,0 +1,31 @@
+{
+  "editor.formatOnSave": true,
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  },
+
+  "[javascript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[typescript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[typescriptreact]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+
```

Comment:
May want to get rid of these since prettier is not configured atm in the ui side, but at the same time that can be a UI problem whenever someone makes a change next

Reply:
yeah will leave it for ui dev